go-backend/db/user_blacklisted_tokens.go

```diff
@@ -17,3 +30,13 @@ func (s *pgStore) CleanBlacklistedTokens() (err error) {
 	}
 	return
 }
+
+func (s *pgStore) CreateUserBlacklistedToken(ctx context.Context, blacklistedToken UserBlacklistedToken) (err error) {
+	_, err = s.db.Exec(insertBlacklistedToken, blacklistedToken.UserID, blacklistedToken.Token, blacklistedToken.ExpirationDate)
```

Comment:
I would suggest using `s.db.MustExec(...)` here instead so we'll be sure to get a failure if that query doesn't execute for some reason. If we're blacklisting a token, we want to make **sure** it no longer has access to do anything, and if that process fails we need it to throw an error so we can get alerted and do something sooner rather than later.

Reply:
The downside of `s.db.MustExec` is if it fails it `panics` and the request is dropped, this is not will not allow us to further process request. i.e we will not be able to return our error response to client. Hence I would prefer `db.Exec` here.